### ISSUE TITLE
Tensorflow on Power9

### ISSUE THREAD

**Issue by golivag** [easybuilders/easybuild-easyconfigs#16010]
Hi there,
I'm trying to build tensorflow on power9 with RHEL8 and get this error:

```
== 2022-08-09 19:31:49,029 run.py:233 INFO running cmd:  pip install --prefix=/opt/software/easybuild/software/TensorFlow/2.7.1-foss-2021b-CUDA-11.4.1  --no-deps  --ignore-installed  --no-index  --no-build-isolation  /opt/software/easybuild/sources/t/TensorFlow/extensions/tensorflow_io_gcs_filesystem-0.24.0-cp39-cp39-manylinux_2_12_x86_64.manylinux2010_x86_64.whl 
== 2022-08-09 19:31:49,793 build_log.py:169 ERROR EasyBuild crashed with an error (at easybuild/base/exceptions.py:124 in __init__): cmd " pip install --prefix=/opt/software/easybuild/software/TensorFlow/2.7.1-foss-2021b-CUDA-11.4.1  --no-deps  --ignore-installed  --no-index  --no-build-isolation  /opt/software/easybuild/sources/t/TensorFlow/extensions/tensorflow_io_gcs_filesystem-0.24.0-cp39-cp39-manylinux_2_12_x86_64.manylinux2010_x86_64.whl" exited with exit code 1 and output:
ERROR: tensorflow_io_gcs_filesystem-0.24.0-cp39-cp39-manylinux_2_12_x86_64.manylinux2010_x86_64.whl is not a supported wheel on this platform.
```

Any help appreciated

**Comment by golivag** [comment]
I'm using TensorFlow-2.7.1-foss-2021b-CUDA-11.4.1.eb

**Comment by branfosj** [comment]
See #15882

**Comment by boegel** [comment]
fixed by #15882, so closing...
